libs/datasets/timeseries.py

```diff
@@ -124,6 +124,15 @@ def get_subset(
 
         return self.__class__(data)
 
+    def get_data_for_fips(self, fips): 
+        pd_data = self.get_subset(aggregation_level=AggregationLevel.COUNTY, fips=fips).data
```

Comment:
fwiw, I don't think this has to be county - if you pass a state fips code, I wouldn't expect it to return none